addons/web/views/webclient_templates.xml

```diff
@@ -100,7 +100,7 @@
             <link rel="stylesheet" href="/web/static/src/css/data_export.css"/>
             <link rel="stylesheet" href="/base/static/src/css/modules.css"/>
 
-            <link rel="stylesheet" href='/website/static/src/less/import_bootstrap.less'/>
+            <link rel="stylesheet" href='/web/static/src/less/import_bootstrap.less'/>
```

Comment:
@kkinateder you have to get rid of any updates non-related to ``set_all_attrs`` issue to get a non-zero chance be merged. You have to make a separate pull request for other updates.